template-files/update-tscircuit-package.yml

```
@@ -0,0 +1,128 @@
+# =============================================================================
+# Do not change anything in this file.
+# This workflow automatically updates @tscircuit packages when triggered by
+# the source repository's trigger-dependent-repo-update.yml workflow.
+# =============================================================================
+
+name: Update @tscircuit Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      package_name:
+        description: 'Package name to update (e.g., @tscircuit/core)'
+        required: true
+        type: string
+        default: '<PACKAGE_NAME>'
+
+env:
+  PACKAGE_NAME: ${{ inputs.package_name }}
+  GITHUB_TOKEN_SECRET: "TSCIRCUIT_BOT_GITHUB_TOKEN"
+  BOT_EMAIL: "bot@tscircuit.com"
+  BOT_NAME: "tscircuitbot"
+
+jobs:
+  update-package:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Validate package name configuration
+        run: |
+          if [ "${{ env.PACKAGE_NAME }}" = "<PACKAGE_NAME>" ] || [ -z "${{ env.PACKAGE_NAME }}" ]; then
+            echo "❌ ERROR: PACKAGE_NAME is not configured properly!"
+            echo "❌ Current value: '${{ env.PACKAGE_NAME }}'"
+            echo "❌ Please set a valid package name (e.g., '@tscircuit/core')"
+            echo "❌ Either:"
+            echo "   1. Configure the default value in this workflow file, or"
+            echo "   2. Pass it as input when triggering manually"
+            exit 1
+          fi
+          echo "✅ Package name validated: ${{ env.PACKAGE_NAME }}"
+      
+      - name: Checkout code
+        uses: actions/checkout@v4
+        
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v2
+        
+      - name: Close existing PRs
+        run: |
+          echo "🔍 Checking for existing update PRs..."
+          gh pr list --repo ${{ github.repository }} --state open --json number,title --jq '.[] | select(.title | contains("update ${{ env.PACKAGE_NAME }}")) | .number' | xargs -I{} gh pr close {} --comment "Closing in favor of a new update PR"
+        env:
+          GH_TOKEN: ${{ secrets[env.GITHUB_TOKEN_SECRET] }}
```

use TSCIRCUIT_BOT_TOKEN without making it part of env

```suggestion
          GH_TOKEN: ${{ secrets[env.GITHUB_TOKEN_SECRET] }}
```